CHANGELOG.md

```diff
@@ -4,11 +4,12 @@
 This release will bump the Realm file format from version 23 to 24. Opening a file with an older format will automatically upgrade it from file format v10. If you want to upgrade from an earlier file format version you will have to use Realm Kotlin v1.13.1 or earlier. Downgrading to a previous file format is not possible.
 
 ### Breaking changes
-* None.
+* Sync progress updates no longer report `transferredBytes` and `totalBytes`. (Issue [#1744](https://github.com/realm/realm-kotlin/issues/1744) [RKOTLIN-1079](https://jira.mongodb.org/browse/RKOTLIN-1079)).
```

Comment:
I guess the new values are the enhancement, but I think I would have combined both into one breaking change entry and put emphasis on the new better precision ... And support for flx-sync, right?